src/orchestrator/unary.rs

```diff
@@ -270,13 +270,21 @@ impl Orchestrator {
                         let ctx = ctx.clone();
                         let detector_id = detector_id.clone();
                         let detector_params = detector_params.clone();
-                        let detector_config = ctx.config.detectors.get(&detector_id).unwrap();
+                        let detector_config = ctx.config.detectors.get(&detector_id).expect(
+                            &format!("error retrieiving config value for key {}", detector_id),
```

Comment:
This has a typo "retrieiving", but I think we should simplify to something like "detector `<detector_id>` not found". Related to prior comment, we should check this and return an error to the user rather than panic.